analyzers/tests/SonarAnalyzer.UnitTest/TestSuiteInitialization.cs

```diff
@@ -17,12 +38,10 @@ public static void AssemblyInit(TestContext context)
             Console.WriteLine(@"Running tests initialization...");
             Console.WriteLine(@$"Build reason: {Environment.GetEnvironmentVariable(TestContextHelper.BuildReason) ?? "Not set / Local build"}");
 
-            var parseOptions = ParseOptionsHelper.GetParseOptionsOrDefault(null).ToList();
-
-            var csVersions = parseOptions.OfType<CSharpParseOptions>().Select(GetVersion);
+            var csVersions = ParseOptionsHelper.OrDefault(null, LanguageNames.CSharp).Cast<CSharpParseOptions>().Select(GetVersion);
```

Comment:
I think that it would be great to have one more method `GetDefaults`. Calling `ParseOptionsHelper.OrDefault(null` is a bit confusing.

Reply:
Done in #5269